apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -67,27 +68,41 @@ static void ble_device_scanned_cb(ble_scanned_device *scanned_device)
 	return;
 }
 
-static void ble_device_disconnected_cb(ble_conn_handle conn_id)
+static void ble_device_disconnected_cb(ble_client_ctx *ctx)
 {
-	RMC_LOG(RMC_TAG, "'%s' is called\n", __FUNCTION__);
+	RMC_LOG(RMC_TAG, "'%s' is called[%p]\n", __FUNCTION__, ctx);
+	(void)sem_post(&g_conn_sem);
 	return;
 }
 
-static void ble_device_connected_cb(ble_device_connected *dev)
+static void ble_device_connected_cb(ble_client_ctx *ctx, ble_device_connected *dev)
 {
 	int ret;
```

Comment:
Please remove unnecessary local variable :)

Reply:
I fixed it